modules/KalturaSupport/components/dualScreen/nativePlayer.js

```diff
@@ -85,7 +85,7 @@
         },
 
         supportsOptimisticSeeking: function supportsOptimisticSeeking() {
-            return !this.isABR();
+            return true;
```

Comment:
@vladm3  
Whats the point having a function that always returns true?

Reply:
@eitanavgil - `nativePlayer.js` and `flashPlayer.js` both inherit from `videoPlayer.js` and this method returns `false` for `flashPlayer.js`